document-generator-api/src/main/java/uk/gov/companieshouse/document/generator/api/service/impl/DocumentTypeServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.companieshouse.document.generator.api.service.impl;
+
+import uk.gov.companieshouse.document.generator.api.Exception.DocumentGeneratorServiceException;
+import uk.gov.companieshouse.document.generator.api.service.DocumentTypeService;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DocumentTypeServiceImpl implements DocumentTypeService {
+
+    @Override
+    public String getDocumentType(String resourceUri) throws DocumentGeneratorServiceException {
+
+        Map<String, String> docType = new HashMap<>();
+        docType.put("ACCOUNTS", "/transactions\\/.*\\/(?:company-)?accounts\\/.*");
```

Comment:
Can the string literal be made a constant for readability when more document types are added?

Reply:
Also, document type "ACCOUNTS" could be enumerated.